JarrydWorland/2021_Capstone_GP2_Team
# Player shooting adjustments

## Description
This PR makes a few adjustments to how the player shoots. It adds a `ProjectileSpawn` point onto the player so that projectiles originate from the correct location on the players sprite. It also adds aim assist for controllers to make aiming easier.

## Changes include
- Added ProjectileSpawn to Player (7f7fff5)
- Added aim assist for controllers (34fd0dd)

## How to test
- Play with both mouse and controller and ensure that when you shoot the projectiles are heading towards where you are aiming correctly.
- On the Players `PlayerWeaponBehaviour` component, you can adjust the `AimAssistConeAngle` to change how much aim assist there is. Setting it to `0` should result in no aim assist while `360` should make you no-scope like a champ.
-  you can also enable a `ShowLockOn` option to show a target on enemies when aim assist is active. Not sure if we should actually use this but I decided to leave it in as an option.
